code/modules/clothing/clothing.dm

```diff
@@ -24,7 +24,7 @@
 
 // Aurora forensics port.

```

Comment:
Believe it or not. In my years as a babikoder, myself and Sue wrote our current forensics system. It was pretty awful and non-readable, but it worked! And then Zuzu refactored and ported it to bay. And then we ported it as a part of the first Baymeme of 2016!

Funny how life works. I now dream of scrapping the entire system and implementing something new in 2018.